Aquality.Selenium/src/Aquality.Selenium/Browsers/Browser.cs

```diff
@@ -134,6 +134,15 @@ public void GoTo(string url)
             Navigate().GoToUrl(url);
         }
 
+        /// <summary>
+        /// Navigates to desired url in new tab.
+        /// </summary>
+        /// <param name="url">String representation of URL.</param>
+        public void OpenUrlInNewTab(string url)
```

Comment:
Just looked at `GoTo(string url)` method - seems that we should not include parameter's name in the method's name.
So the current method should be named `OpenInNewTab(string url)`?

Reply:
Shouldn't we have this method inside BrowserTabNavigation?

Reply:
I made like this, because OpenInNewTab method, in my opinion, more similar with GoToUrl method, so related to Browser but not with tab management.

Reply:
But to close this new tab you have to user BrowserTabNavigation, so it looks like BrowserTabNavigation is more relevant place for this method.

Reply:
if you both think so, okay
fixed